sections/9-affiliated.tex

```diff
@@ -12,9 +12,9 @@ \section{Affiliated Packages}
 \end{itemize}
 Developers can formally apply to become an affiliated package to the lead developer and final approval is required by the SunPy board.
 Packages are re-reviewed on a yearly basis to ensure that they continue to meet the standards.
-All affiliated packages are listed on \url{sunpy.org}, are provided support by the SunPy developer community, and advertised at conferences and workshops.
+All affiliated packages are advertised on the website\footnote{\url{https://sunpy.org/team\#affiliated-packages}}, are provided support by the SunPy developer community, and advertised at conferences and workshops.
```

Comment:
```suggestion
All affiliated packages are advertised on the website\footnote{\url{https://sunpy.org/team#affiliated-packages}}, are provided support by the SunPy developer community, and advertised at conferences and workshops.
```

Reply:
I need to escape this character to get the build to work.

Reply:
Ah good point. I forgot.